CRM/Upgrade/Incremental/sql/4.7.32.mysql.tpl

```diff
@@ -1 +1,9 @@
 {* file to handle db changes in 4.7.32 during upgrade *}
+
+-- TODO Issue number - Add Membership Types to list of entities that can use custom data via UI.
+SELECT @option_group_id_cg := MAX(id) FROM `civicrm_option_group` where name = 'cg_extend_objects';
+SELECT @option_group_cg_weight := MAX(weight) FROM `civicrm_option_value` where option_group_id = @option_group_id_cg;
+
+INSERT INTO `civicrm_option_value`
+(`option_group_id`, {localize field='label'}`label`{/localize}, `value`, `name`, `grouping`, `filter`, `is_default`, `weight`, `description`, `is_optgroup`, `is_reserved`, `is_active`, `component_id`, `domain_id`, `visibility_id`, `icon`, `color`)
+VALUES(@option_group_id_cg, 'Membership Types', 'MembershipType', 'civicrm_membership_type', NULL, 0, 0, (@option_group_cg_weight + 1), NULL, 0, 0, 1, NULL, NULL, NULL, NULL, NULL);
```

Comment:
@eileenmcnaughton Is this the right way to extend the UI to support more custom data entities?  I note that quite a few are hardcoded but a couple are added via cg_extend_objects so I've done the same with MembershipType.

Reply:
@seamuslee001 Any idea what is causing the build failure as it seems to be related to this DB upgrade

Reply:
@mattwire Matt i think you need to put `{localize}'{ts escape="sql"}` and {/ts}{/localize}` around the Membership Types label as well.  Apart from that not sure what else. Also this PR will need to be rebased